ingestion/examples/sample_data/glue/tables.json

```diff
@@ -0,0 +1,9254 @@
+{
+  "tables": [
+    {
+      "id": "5866244a-1a5a-433f-8807-71ae2a2ee9bb",
+      "name": "dim_address",
```

Comment:
we are adding the tables with same name for Glue. Lets add few tables but distinct, so if someone loads sample_data it doesn't look like we have duplicated metadata

Reply:
Fixed